src/components/BookDisplay.js

```diff
@@ -3,6 +3,7 @@ import { useSelector, useDispatch } from 'react-redux';
 import BookContainer from './BookContainer';
 import Books from './Books';
 import { fetchBooks } from '../redux/books/booksSlice';
+import '../styles/BookDisplay.css';
```

Comment:
- [ ] it would be better to add the vertical line in the book card in order to match the Design 
<img width="243" alt="Screenshot 2023-05-19 at 16 37 01" src="https://github.com/Bellagirl-maker/book-store/assets/61790587/40205ceb-e445-47da-bd6b-706c3f585c5e">

Reply:
@youmari Done with this change